kvmagent/kvmagent/plugins/bmv2_gateway_agent/object.py

```
@@ -73,7 +87,14 @@ class BmInstanceObj(Base):
     @classmethod
     def from_json(cls, req):
         obj = cls()
-        obj.construct(obj.body(req).get('bmInstance', {}))
+        instance = obj.body(req).get('bmInstance', {})
+        obj.construct(instance)
+
+        setattr(obj, 'extra_provision_nic_infos', [])
```

Ruff (0.14.13)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)